TW 107 and 108 (#1884)
temporalio/documentation: docs-src/concepts/what-is-the-temporal-web-ui.md

### Namespace selection

> The Namespace Switcher (located in the left navigational menu) allows users to search for or select a Namespace.
> After a Namespace is selected, the user is redirected to that Namespace's Recent Workflows.
> In Temporal Cloud, users can only access Namespaces that they have been granted access to.

Reviewer: ```suggestion
You can search for or select a Namespace by using the Namespace Switcher, located in the left navigational menu.
After you select a Namespace, the Web UI shows the Recent Workflows page for that Namespace.
In Temporal Cloud, users can access only the Namespaces that they have been granted access to.
```

### Recent Workflows

Recent Workflows lists all Workflow Executions run in the past 24 hours.
The default number shown is 1,000 Workflow Executions.

Users can list Workflow Executions by any of the following:

> - [Workflow ID](/concepts/what-is-a-workflow-id)
> - [Workflow Type](/concepts/what-is-a-workflow-type)
> - Status

Reviewer: ```suggestion
- Status
- [Workflow ID](/concepts/what-is-a-workflow-id)
- [Workflow Type](/concepts/what-is-a-workflow-type)
- Start time
- End time
```
Sequence updated to match UI.

- A [List Filter](/concepts/what-is-a-list-filter)

> Date and Time—users can set their preferred date and time format as one of the following:

Reviewer: ```suggestion
For start time and end time, users can set their preferred date and time format as one of the following:
```

### Import Event History

> Allows the Event History to be imported in JSON format for populating a Workflow details page.

Reviewer: I'm not sure whether this feature is available via Web UI anymore.

Author: Went and checked, and sure enough, it is not! I'll go change this.

### Data Encoder

The Web UI can use the same Data Converter that Workers use for inputs and return values.

> The UI supports both a Remote Codec endpoint and/or the `tctl` plugin port.

Reviewer: ```suggestion
The UI supports both a Remote Codec endpoint and the `tctl` plugin port.
```
